sdk/enter.go

```diff
@@ -200,7 +200,8 @@ func (e *enter) MountGnupgHome() error {
 	// gpg gets confused when GNUPGHOME isn't ~/.gnupg, so mount it there.
 	// Additionally, set the GNUPGHOME variable so commands run with sudo
 	// can also use it.
-	newHome := filepath.Join(e.Chroot, e.User.HomeDir, ".gnupg")
+	newHomeInChroot := filepath.Join("home", e.User.Username, ".gnupg")
```

Comment:
Won't this string be a relative path?  Is that a problem when using it directly for `GNUPGHOME` below?

Reply:
Yeah, good catch. I suspect I would've hit this when I reached the signing part of the release process. Fixed!